src/components/breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import './Breadcrumb.css';
+import breadcrumbData from './constants/breadcrumbData'
+
+function Breadcrumb(){
+
+    const {home, city, layout, shop} = breadcrumbData;
+
+    return (
+        <div className="breadcrumb">
```

Comment:
Can you tweak the data in such a way that you don't have to repeatedly type it out instead render a list